test/Common/ClientFunctions.cs

```diff
@@ -3,14 +3,15 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.ContextInterfaces;
 
 namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
 {
     internal static class ClientFunctions
```

Comment:
I would add two more sample Functions here. One for `IDurableOrchestrationClient`, and one for `IDurableEntityClient`. Then we can add a test for each that makes sure that we are properly binding to these interfaces as well.